src/App.js

```diff
@@ -14,6 +15,9 @@ function App() {
         <Switch>
           <Route path="/dragons" component={Dragons} />
           <Route path="/rockets" component={Rockets} />
+          <Route path="/missions">
+            <Missions />
+          </Route>
```

Comment:
You can use the prop component from the Route to render the component that will look cleaner instead of passing the component as a child, like: <Route path="/missions" component={Missions} />

Reply:
Done!